packages/rspack-test-tools/src/processor/stats.ts

```diff
@@ -171,7 +168,7 @@ export class StatsProcessor<
 		actual = actual
 			.replace(/\r\n?/g, "\n")
 			// CHANGE: Remove potential line break and "|" caused by long text
-			.replace(/((ERROR|WARNING)([\s\S](?!╭|├))*?)(\n  │ )/g, "$1")
+			.replace(/((ERROR|WARNING)([\s\S](?!╭|├))*?)(\n {2}│ )/g, "$1")
```

Comment:
Is `{2}`  appended by biome ?
I don't quite understand this place.

Reply:
This is because the rule `noMultipleSpacesInRegularExpressLiterals` prohibits two consecutive spaces.